docs/docs/cmd/spo/site/site-admin-list.mdx

```diff
@@ -28,7 +28,7 @@ m365 spo site admin list [options]
 
 :::info
```

Comment:
Let's remove the last line of this admonition since it's not really worth mentioning anymore.